src/Mappers/ElementIndexMapper.php

```diff
@@ -45,8 +45,7 @@ public function export(array $elementTypes): array
         foreach ($elementTypes as $elementType) {
             $settings = Craft::$app->elementIndexes->getSettings($elementType);
             if (is_array($settings)) {
-                $elementTypeName = str_replace('craft\\elements\\', '', $elementType);
```

Comment:
Can we fall back on craft\elements to make it backwards compatible?

Reply:
Added backwards compatibility